rust/foxglove/src/typed_channel.rs

```diff
@@ -57,13 +59,21 @@ impl<T: Serialize + JsonSchema> TypedMessage for T {
     }
 }
 
-/// A typed [`Channel`] for messages that implement [`TypedMessage`].
-pub struct TypedChannel<T: TypedMessage> {
+/// A typed [`Channel`] for messages that implement [`Encode`].
+/// Channels are immutable, returned as `Arc<Channel>` and can be shared between threads.
```

Comment:
Add a newline to keep the front-page "Structs" listing concise.

```suggestion
/// A typed [`Channel`] for messages that implement [`Encode`].
///
/// Channels are immutable, returned as `Arc<Channel>` and can be shared between threads.
```